assets/scss/core/_reset.scss

```diff
@@ -25,6 +25,22 @@ RESET DOPPLER STYLE GUIDES
   box-sizing: border-box;
 }
 
+::-webkit-scrollbar-thumb {
+  background: rgba(0, 0, 0, 0.3);
+  border-radius: 10px;
+}
+
+::-webkit-scrollbar-track {
+  background: rgba(0, 0, 0, 0.1);
+  border-radius: 10px;
+  margin: 5px;
+}
+
+::-webkit-scrollbar {
```

Comment:
Does this changes only affect webkit? is that fine?

Reply:
I agree with this, why are you styling webkit scrollbar only? What about the other browsers? Maybe I'm missing something.

Reply:
It seems that this _Doppler's issue_ only affects webkit, so, it is fine fixing webkit.